### DISCUSSION TITLE
How to Pass Edit Method data to Render Method

### DISCUSSION THREAD

**Discussion by tmgomas** [livewire/livewire#3759] [1. Help]
I am Working with small project and both are working fine . My issue is i am tring to get wire.click data to Edit Component

```
public function UpdateFeeScollector(SchoolFeeCollector $SchoolFeeCollectorID)
    {
        $this->showEditModal = true;
        $this->SchoolFeeCollectorID = $SchoolFeeCollectorID;
        $this->fc_fee_scheem_id   = $SchoolFeeCollectorID->fc_fee_scheem_id;
        $this->fc_fee_group_id   = $SchoolFeeCollectorID->fc_fee_group_id;
        $this->fc_cf_amount   = $SchoolFeeCollectorID->fc_cf_amount ;
        $this->dispatchBrowserEvent('show-form');
    }
```
Its Getting data to and passing value to view and also select option value but it not showing in selected value … But same component I am passing to selected option and showing perfect Using Render Method How to I pass this value to Render Method

**Comment by devcircus** [comment]
Be sure anything you need access to in the view, that's not passed via the render method, has a public property on the component which holds the value you need.

**Reply by tmgomas** [reply]
> Be sure anything you need access to in the view, that's not passed via the render method, has a public property on the component which holds the value you need.

Firstly Thank for your advice . But When send the public property  select 2 now showing it to selected value
